### PR TITLE
Fix Issue 19152 - 2.081.1 Regression - Seg fault in traits.d

### PR DESCRIPTION
**Pull request by JinShil**

### CODE REVIEW

**Comment by JinShil** [comment]
What's going on with `src/core/sys/posix/sys/statvfs.d` and GDC?  It was modified in https://github.com/dlang/druntime/pull/2265, but that had targeted `master`, not `stable`.  This PR is targeting `stable`

cc @ibuclaw @jmdavis @wilzbach

**Comment by jmdavis** [comment]
It looks like it can't handled enum members being deprecated, which is a very recent feature.

**Review by jacob-carlborg — APPROVED** [review]

File: test/compilable/test19152.d [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// https://issues.dlang.org/show_bug.cgi?id=19152
+
+class TBasicAction
+{
+    int OnExecute()
+    {
+        auto name = __traits(getOverloads, this, "OnExecute").stringof;
+        return 0;
+    }
+}
```
Comment (jacob-carlborg) [inline test/compilable/test19152.d]: Is there an existing file this can be added to? This will reduce the total execution time of the tests.

Comment (JinShil) [reply test/compilable/test19152.d]: I couldn't find an obvious fit in `test/compilable`.  Shall I start a new file called `traits.d` to contain all future traits-related tests.

Comment (jacob-carlborg) [reply test/compilable/test19152.d]: That's fine by me.

**Comment by JinShil** [comment]
Looks like LDC is giving the same error now in Semaphore CI.  Is the CI using the correct druntime code?

**Review by RazvanN7 — APPROVED** [review]

**Comment by wilzbach** [comment]
>  Is the CI using the correct druntime code?

Doesn't look like it. https://github.com/dlang/dmd/pull/8560 should fix this.

**Review by wilzbach — APPROVED** [review]

**Pull request merged**